### ISSUE TITLE
Color contrast of focused Menu Item does not meet 4.5:1 in Northstar Menu

### ISSUE THREAD

**Issue by davyrwoo** [microsoft/fluentui#14538]
### Environment Information

- **Package version(s)**: 0.51.0
- **Browser and OS versions**: Chrome

### Describe the issue:

Color contrast of text and background of the focused menu item does not meet 4.5:1 ratio in Northstar Menu with `underlined `and `primary`. The ratio instead is 1.711:1

![Menu item color contrast](https://user-images.githubusercontent.com/56053686/90292783-71adbb00-de37-11ea-9e8e-9bcb7c7f9145.PNG)

### Please provide a reproduction of the issue in a codepen:

https://codesandbox.io/s/o5ql0?module=/example.js

#### Actual behavior:

1. Tab to menu item
2. Observe the contrast ratio for the focused text and background is 1.711:1

#### Expected behavior:

1. Tab to menu item
2. Contrast ratio should be at least 4.5:1

### Documentation describing expected behavior

https://www.w3.org/WAI/WCAG21/Techniques/general/G18

**Comment by khmakoto** [comment]
@jurokapsiar fyi

**Comment by jurokapsiar** [comment]
fixed in https://github.com/microsoft/fluentui/pull/14525
